.github/workflows/spriggit-out.yml

```diff
@@ -0,0 +1,54 @@
+name: spriggit-out
+on:
+  schedule:
+    - cron: '0 9 * * *'
+  workflow_dispatch:
+jobs:
+  publish:
+    permissions: write-all
```

Comment:
Yeah, not sure what it would need either, and it was failing one of the steps without.  Fine with narrowing it down, but maybe as a followup once everything settles?

Reply:
Agreed!